content/docs/command-reference/remote/add.md

```diff
@@ -344,6 +344,21 @@ $ dvc remote add -d myremote https://example.com/path/to/dir
 
 <details>
 
+### Click for WebDAV
+
+```dvc
+$ dvc remote add -d myremote webdavs://example.com/public.php/webdav
+```
```

Comment:
We just realized that PHP is kind of a weird example to have (see https://github.com/iterative/dvc.org/pull/1695#pullrequestreview-472130005).

We'll be reviewing these sample URLs in #1706, in case you're interested @iksnagreb 🙂